dev-libs/libclangformat-ide/files/clang-qtcreator-compat.patch

```diff
@@ -0,0 +1,185 @@
+From fa1b9053729ec6a4425a44ec5502dd388928274a Mon Sep 17 00:00:00 2001
+From: Ivan Donchevskii <ivan.donchevskii@qt.io>
+Date: Fri, 12 Oct 2018 08:19:25 +0200
+Subject: [PATCH] [clang-format] Introduce the flag which allows not to shrink
```

Comment:
have you asked the gentoo llvm project if they're willing to take the patch, possibly behind a USE flag?

Reply:
https://bugs.gentoo.org/722892